src/index.pug

```diff
@@ -226,28 +226,28 @@ html
 
     footer
       .container
-        ul.footer-links 
-          li 
+        ul.list-unstyled.footer-links.row.justify-content-center
+          li.col-auto
```

Comment:
I'd rather be a bit explicit about it, and yeah, it's good practice as a reminder that this is handled by the grid.